src/test/java/cz/cvut/kbss/termit/service/config/ConfigurationProviderTest.java

```diff
@@ -21,19 +19,16 @@
     private Configuration config;
 
     @Autowired
-    private UserRoleRepositoryService service;
+    private Environment environment;
 
+    @Autowired
     private ConfigurationProvider sut;
 
-    @BeforeEach
-    void setUp() {
-        this.sut = new ConfigurationProvider(config, service);
-    }
-
     @Test
     void getConfigurationReturnsConfigurationDtoWithRelevantConfigurationValues() {
         final ConfigurationDto result = sut.getConfiguration();
         assertNotNull(result);
         assertEquals(config.getPersistence().getLanguage(), result.getLanguage());
+        assertEquals(environment.getProperty("spring.servlet.multipart.max-file-size"), result.getMaxFileUploadSize());
```

Comment:
Why not configure it from test/application.yml through Configuration

Reply:
See below.